pkgs/applications/misc/robomongo/default.nix

```diff
@@ -1,29 +1,50 @@
-{ stdenv, fetchurl, qtbase, openssl, boost, cmake, scons, python, pcre, bzip2 }:
+{ stdenv, fetchurl, zlib, glib, xorg, dbus, fontconfig, freetype, xkeyboard_config, makeWrapper }:
 
 stdenv.mkDerivation {
-  name = "robomongo-0.8.4";
+  name = "robomongo-0.9.0";
 
   src = fetchurl {
-    url = https://github.com/paralect/robomongo/archive/v0.8.4.tar.gz;
-    sha256 = "199fb08701wrw3ky7gcqyvb3z4027qjcqdnzrx5y7yi3rb4gvkzc";
+    url = "https://download.robomongo.org/0.9.0/linux/robomongo-0.9.0-linux-x86_64-0786489.tar.gz";
```

Comment:
I've not tried to build version 0.9.0 from source since they provide a binary with all the necessary libraries in a tar.gz.